src/sentry/lang/native/minidump.py

```diff
@@ -96,6 +105,65 @@ def merge_process_state_event(data, state, cfi=None):
     data.setdefault('debug_meta', {})['images'] = images
 
 
+def merge_attached_event(mpack_event, data):
+    # Merge msgpack serialized event.
+    if sys.getsizeof(mpack_event) > MAX_MSGPACK_EVENT_SIZE_BYTES:
```

Comment:
just make this `len(mpack_event)` :)

Reply:
This one won't work: `TypeError: object of type '_io.BytesIO' has no len()`